docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -141,6 +141,11 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -XX:PerBytecodeRecompilationCutoff=10000
     -Djdk.attach.allowAttachSelf=true
     -Djdk.nio.maxCachedBufferSize=2000000
+    -XX:+UnlockDiagnosticVMOptions
+    -XX:+UseAESCTRIntrinsics
+
+.. note::
```

Comment:
Let’s move this to the bottom of this section, directly after the temporary directory paragraph, and remove the note formatting so that it’s a normal sentence/paragraph ending in a period. We don’t need to draw attention to it the way a note does.

Reply:
done